draft-wussler-openpgp-pqc.md

```diff
@@ -401,47 +338,33 @@ abbreviation "PQ/T" is used. The short form "PQ(/T)" stands for PQ or PQ/T.
-on which the two CRYSTALS-* schemes and SPHINCS+ are based, are fundamentally
+on which the two ML-* schemes and SLH-DSA are based, are fundamentally
 different, and accordingly the level of trust commonly placed in them as well
 as their performance characteristics vary.
 
-\[Note to the reader: This specification refers to the latest NIST submission
-papers of each scheme as if it were a specification. This is a temporary
-solution that is owed to the fact that currently no other specification is
-available. The goal is to provide a sufficiently precise specification of the
-algorithms already at the draft stage of this specification, so that it is
-possible for implementers to create interoperable implementations. As soon as
-standards by NIST or the IETF for the PQC schemes employed in this
-specification are available, these will replace the references to the NIST
-submission papers. Furthermore, we want to point out that, depending on
-possible changes to the schemes standardized by NIST, this specification may be
-updated substantially as soon as corresponding information becomes available.\]
-
-### CRYSTALS-Kyber {#kyber-intro}
-
-CRYSTALS-Kyber [Kyber-Subm] is based on the hardness of solving the
-learning-with-errors problem in module lattices (MLWE). The scheme is believed
-to provide security against cryptanalytic attacks by classical as well as
-quantum computers. This specification defines CRYSTALS-Kyber only in composite
-combination with ECC-based encryption schemes in order to provide a pre-quantum
-security fallback.
-
-### CRYSTALS-Dilithium {#dilithium-intro}
-
-CRYSTALS-Dilithium, defined in [DILITHIUM-Subm], is a signature scheme that,
-like CRYSTALS-Kyber, is based on the hardness of solving lattice problems in
-module lattices. Accordingly, this specification only defines
-CRYSTALS-Dilithium in composite combination with ECC-based signature schemes.
-
-### SPHINCS+
-
-SPHINCS+ [SPHINCS-Subm] is a stateless hash-based signature scheme. Its
-security relies on the hardness of finding preimages for cryptographic hash
-functions. This feature is generally considered to be a high security
-guarantee. Therefore, this specification defines SPHINCS+ as a standalone
-signature scheme.
-
-In deployments the performance characteristics of SPHINCS+ should be taken into
+### ML-KEM {#mlkem-intro}
+
+ML-KEM [FIPS-203] is based on the hardness of solving the learning-with-errors
+problem in module lattices (MLWE). The scheme is believed to provide security
+against cryptanalytic attacks by classical as well as quantum computers. This
+specification defines ML-KEM only in composite combination with ECC-based
+encryption schemes in order to provide a pre-quantum security fallback.
+
+### ML-DSA {#mldsa-intro}
+
+ML-DSA [FIPS-204] is a signature scheme that, like ML-KEM, is based on the
+hardness of solving lattice problems in module lattices. Accordingly, this
```

Comment:
I think the reason why we phrased it differently is because the security of ML-DSA bases on different problems (MLWE and MSIS). See §3.2 of FIPS-204. I guess there is space for optimization.

Reply:
@sehlen Are you fine with this formulation? FIPS-204 states that security goes back to a non-standard version of MSIS.